File: HackerEarth/Basic Programming/Basics of Implementation/the_great_kian.py
n = raw_input()
x = raw_input().split()
#print x
sum1 = 0
sum2 = 0
sum3 = 0

for i in range(int(n)):
    if int(x[i]) % 3 == 1:
        sum1 = sum1 + int(x[i])
    if int(x[i]) % 3 == 2:
        sum2 = sum2 + int(x[i])
    if int(x[i]) % 3 == 0:
        sum3 = sum3 + int(x[i])

print str(sum1) + " " + str(sum2) + " " + str(sum3)

File: HackerEarth/Basic Programming/Basics of Implementation/the_great_kian.cpp
#include <iostream>
#include <vector>
using namespace std;

int main()
{
    unsigned long long i, j, n;
    cin >> n;
    unsigned long long sum1, sum2, sum3;
    sum1 = sum2 = sum3 = 0;

    vector<unsigned long long> c(n);

    for(i = 0; i < n; i++){
        cin >> c[i];

        if(i % 3 == 1){
            sum1 = sum1 + c[i];
        }
        if(i % 3 == 2){
            sum2 = sum2 + c[i];
        }
        if(i % 3 == 0){
            sum3 = sum3 + c[i];
        }
    }

    cout << sum3 << " " << sum1 << " " << sum2 << endl;

    return 0;
}
